deps/rabbitmq_management/priv/www/js/oidc-oauth/helper.js

```diff
@@ -4,18 +4,24 @@ var _management_logger;
 
 function oauth_initialize_if_required() {
     rabbit_port = window.location.port ? ":" +  window.location.port : ""
-    rabbit_base_uri = window.location.protocol + "//" + window.location.hostname + rabbit_port
+    rabbit_base_uri = window.location.protocol + "//" + window.location.hostname
+      + rabbit_port + rabbit_path_prefix()
 
     var request = new XMLHttpRequest();
-    request.open('GET', rabbit_base_uri + '/api/auth', false);
+    request.open("GET", rabbit_base_uri + "/api/auth", false);
     request.send(null);
     if (request.status === 200) {
         return oauth_initialize(JSON.parse(request.responseText));
-    }else {
+    } else {
         return { "enabled" : false };
     }
 
 }
+function rabbit_path_prefix() {
+  paths = window.location.pathname.split("/")
+  return paths[1] === "js" || paths[1] === "" ? "" : "/" + paths[1]
```

Comment:
Why only first part is used? According to [documentation](https://www.rabbitmq.com/management.html#path-prefix) `anagement.path_prefix setting allows an arbitrary prefix to be set`, but that code for that pull request page returns only `/rabbitmq`, but should get all path
![image](https://user-images.githubusercontent.com/12235437/193770711-aab961cc-3a7c-423d-998d-85df75d3e56e.png)

Reply:
The prefix is not limited to a single segment, that's correct.